### PR TITLE
Resolved tests failures due to change in swagger spec

### PR DESCRIPTION
**Pull request by ajay-sainy**

### CODE REVIEW

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks like a required parameter is being renamed here which is a breaking change. Please bump the Nuget package version to 2.0.0 and update PackageReleaseNotes
Also update the AssemblyVersion and AssemblyFileVersion in AssemblyInfo.cs

**Comment by ajay-sainy** [comment]
Made the requested change.

File: src/SDKs/ManagedServices/Management.ManagedServices/Microsoft.Azure.Management.ManagedServices.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.ManagedServices</PackageId>
     <Description>Microsoft Azure Managed Services Management Library</Description>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
```
Comment (shahabhijeet) [inline src/SDKs/ManagedServices/Management.ManagedServices/Microsoft.Azure.Management.ManagedServices.csproj]: No need to version your tests.
So in your next PR remember you don't need to update test project version

**Review by shahabhijeet — APPROVED** [review]

File: src/SDKs/ManagedServices/Management.ManagedServices/Microsoft.Azure.Management.ManagedServices.csproj (hunk 1 above)
Comment (ajay-sainy) [reply src/SDKs/ManagedServices/Management.ManagedServices/Microsoft.Azure.Management.ManagedServices.csproj]: Thanks for the feedback @shahabhijeet . Pushed the suggested changes.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**